doc/docs.md

```diff
@@ -685,6 +684,70 @@ for i in 0 .. 1000 {
 Note: The above code uses a [range `for`](#range-for) statement and a
 [push operator (`<<`)](#array-operations).
 
+#### Array Types
+
+An array can be of these types:
+| Types        | Example Definition    |
+| ------------ | --------------------- |
+| Number       | `[]int,[]i64`         |
+| String       | `[]string`            |
+| Rune         | `[]rune`              |
+| Boolean      | `[]bool`              |
+| Array        | `[][]int`             |
+| Struct       | `[]MyStructName`      |
+| Channel      | `[]chan`              |
+| Function     | `[]func`              |
+| Interface    | `[]MyInterfaceName`   |
+| Sum Type     | `[]MySumTypeName`     |
+| Generic Type | `[]<T>`               |
+| Map          | `[]map`               |
+| Enum         | `[]enum`              |
+| Alias        | `[]MyAliasTypeName`   |
+| Thread       | `[]xx`                |
+| Reference    | `[]&f64`              |
+| Shared       | `[]xx`                |
```

Comment:
That would typically be something like `[]shared MyStructType`.

Reply:
When listing the possible array types in my last comment my intention was actually not to have them included here. On the other hand there is no such short list of all possible types in V - despite this documentation having 2 sections titles "Types"... So maybe this could stay here for now.

Reply:
I also like the very dry and short way this is handled in the golang specifications.